api-reference/10 UI Components/dxStepper/1 Configuration/onItemContextMenu.md

```diff
@@ -4,28 +4,28 @@ type: function(e)
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+A function that is executed when an item (step) is right-clicked or pressed.
```

Comment:
I guess the word "pressed" by itself can be confusing. 

For example, `onmousedown` event descriptions usually use the word "press". 

https://developer.mozilla.org/en-US/docs/Web/API/Element/mousedown_event

> The `mousedown` event is fired at an [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element) when a pointing device button is **pressed** while the pointer is inside the element.

So I would at least say "long-pressed", because that's the gesture on touch screens for context menus. 

```suggestion
A function that is executed when an item (step) is right-clicked or long-pressed.
```